packages/fxa-shared/express/index.ts

```diff
@@ -2,12 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-module.exports = () => {
-  const cors = require('./cors');
-  const routing = require('./routing');
+import cors from './cors';
+import routing from './routing';
 
-  return {
-    cors,
-    routing,
-  };
-};
+const express = () => ({
```

Comment:
I don't see the point of this function. I guess it maintains backwards compatibility... but maybe this is a chance to get rid of this kind of syntax. 

This would look more like standard typescript if it just exported directly from the other typescript files.

i.e.

```
export * from './cors'
export * from './routing'
```

Reply:
> maintains backwards compatibility...

Yeah.  It's almost certain that we'd need to update/refactor bunch of tests depending how these are mocked/proxyquire'd.  I didn't particularly want that in this PR.

Reply:
Sounds good. Maybe file a follow up ticket for cleaning this up?

Reply:
Filed https://mozilla-hub.atlassian.net/browse/FXA-6398